app/pages/reservation/reservation-information/InternalReservationForm.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import PropTypes from 'prop-types';
+
+import injectT from '../../../i18n/injectT';
+import FormTypes from '../../../constants/FormTypes';
+
+class UnconnectedInternalReservationForm extends Component {
```

Comment:
If you decide to make this as Fields instead of small form under big form.
Rename the file into `InternalReservationFields`

Reply:
Renamed UnconnectedInternalReservationForm to InternalReservationFields.